dev-libs/c_blosc/patches/c_blosc-1.14.3.patchset

```diff
@@ -0,0 +1,25 @@
+From d014d1b144f867dbfc158eea1f45d26f9a35bf58 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Thu, 7 Dec 2017 21:27:35 +0100
+Subject: [PATCH] Build fix
+
+---
+ tests/test_common.h | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
```

Comment:
Suggestion:
```diff
-Subject: [PATCH] Build fix
+Subject: Build fix
 
----
- tests/test_common.h | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
```
I guess you are using `git format-patch` to create the patchset.

The suggestion above is the result of `haikuporter -e c_blosc` (`-e` is the short option equivalent to `--extract-patchset`).
Since most (if not all) contributors use haikuporter's `-e` option (which we call after a `-B` with a copy of the patchset from the previous release), it would be nice if you also used it.
(I guess the recommended method to create patchsets used in recipes is to use haikuporter.)

If you would like to try, the process is the same, except that, in the end, instead of calling `git format-patch` you call `haikuporter -e c_blosc`.